spec/menu_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'menu'
+
+describe Menu do
+
+    menu = Menu.new
+    item1 = Item.new(1, "Eggs and Turnips", 10)
+    item2 = Item.new(2, "Cheese, Carrot and Marzipan", 12)
+    item3 = Item.new(3, "Pear, Pork and Parmisan", 20)
+    item4 = Item.new(4, "Cabbage on Toast", 90)
+    menu.instance_variable_set(:@menu_items, [item1, item2, item3, item4])
+
+
+    it 'initialises without error' do
+        expect{subject}.not_to raise_error
+    end
+
+    describe 'build_menu' do
```

Comment:
After describe you could have let(:subject) { subject.build_menu)
Then you don't have to repeat yourself